### DISCUSSION TITLE
Make it so objects don't require localization.

### DISCUSSION THREAD

**Discussion by Jdance-Media** [SmartlyDressedGames/Unturned-3.x-Community#4782] [Ideas]
My main point is: "why not?"

People would still be able to use localization for objects. It is clear that objects don't rely on the localized names much. And foliage assets don't have localization either (some people have a bone to pick with you on that one).

**Comment by SDGNelson** [comment]
Fair :P

**Comment by DanielWillett** [comment]
Would be cool to be able to add default localization to any .asset files so they don't all need their own folders but the functionality can stay

ex:
```properties
Metadata
{
    GUID 21594168c1564e3d807ecb178453b374
    Type SDG.Unturned.ObjectAsset, Assembly-CSharp
}
Asset
{
    Type Medium
    
    Rubble Destroy
    Rubble_Reset 60
    Rubble_Health 1
    Rubble_Effect 38
    
    Chart Ignore

    Has_Clip_Prefab false
    Exclude_From_Level_Batching

    LOD None
    Bundle_Override_Path Objects/WhateverThisObjectIs
}
Localization
{
    English
    {
        Name Localized Object Name
    }
    OtherLanguagesMaybeIdk
    {
        Name etc
    }
}
```

**Reply by Jdance-Media** [reply]
I agree. It would be great to integrate localization into the .asset files. I've noticed that some people have switched to that format more recently.
